src/style.scss

```diff
@@ -40,20 +124,24 @@
     -webkit-align-items: center;
     justify-content: center;
     -webkit-justify-content: center;
-    background-color: #E0E0E0;
+    background-color: $input-button-background-color;
```

Comment:
when using variables I think it's better to use `background` instead of `background-color` so to give more power to the user (`background: $color` works exactly the same).

very very iper minor